tests/units/cli/test_config.py

```diff
@@ -359,6 +349,128 @@ def test_save_config(self):
         # Delete the generated JSON config directory
         shutil.rmtree(json_config_dir)
 
+    def test_save_config_selected_tap(self):
+        """Test config target and tap JSON save functionalities if specific taps are selected"""
+        json_config_dir = './pipelinewise-test-config'
+        config = self._get_config(json_config_dir, yaml_path='test_yaml_config_selected_tap')
```

Comment:
I'm wondering why the need for a new folder with taps `test_yaml_config_selected_tap` and not just use the existing ones under `test_import_command`?

Reply:
- [ ] fix it